src/xenia/kernel/xthread.h

```diff
@@ -105,6 +105,8 @@ static_assert_size(X_KTHREAD, 0xAB0);
 
 class XThread : public XObject, public cpu::Thread {
  public:
+  // Burnout Paradise needs proper handle value for certain calculations
```

Comment:
This isn't very descriptive for the future, would be better to reword the PR text in the commit, with the calculation, and the 0x8A/0x82 part.